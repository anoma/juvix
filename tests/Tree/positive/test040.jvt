-- ByteArray

type list {
  nil : list;
  cons : * -> list -> list;
}

function main() : * {
  save(bytearray-from-list-uint8(alloc[cons](258u8, alloc[cons](1u8, alloc[nil]())))) {
    seq(trace(bytearray-length(tmp[0])), tmp[0])
  }
}
