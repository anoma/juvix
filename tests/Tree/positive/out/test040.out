2
alloc[cons](2u8, alloc[cons](1u8, alloc[nil]()))
